breakdown-guides.rst

```diff
@@ -0,0 +1,400 @@
+
+	See :ref:`Risk-Impact-Table` for categories.
+
+Likelihood
+	Overall chance of risk being realized.
+
+	See :ref:`Likelihood-Table` for categories.
+
+Existential Risk
+	Select ``Yes`` if the risk is existential to NOIRLab.
+
+	The Rubin Observatory Risk and Opportunity Board will determine if this is appropriate.
+
+Schedule/Cost Impact Description
+	Use these sections to describe and comment on impact categorization and the decision on rating.
+
+
+.. _Breakdown-Risk-Score-Quantitative:
+
+Risk score and quantitative analysis 
+------------------------------------
+
+The fields under ``Risk Score`` are automatically generated based on input selections from :ref:`risk impacts <Breakdown-Risk-Initial-Impact>`.
+Categories and values are based on the :ref:`Breakdown-Risk-Tool-Tables`.
+The ``Analyze Risk Quantitative`` section will not affect values and categories; however, the section will record the impact justification and provide information needed to categorize the impacts.
+
+.. figure:: /_static/Risk-Example-Analyze-Risk-Score-and-Quantitative.png
+    :name: Risk-Example-Analyze-Risk-Score-and-Quantitative
+
+    Risk Score and Analysis Quantitative sections using an example risk.
+
+Impact Severity
+	Automatically generated category based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.
+
+	See :ref:`Risk-Impact-Table` for categories.
+
+Impact Score
+	Automatically generated value based on ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.
+
+Likelihood Score
+	Automatically generated value based on ``Likelihood``.
+
+Probability
+	Automatically generated value based on ``Likelihood``.
+
+Initial Risk Score
+	Automatically generated value based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.
+
+Minimum Delay (Months)
+	Minimum delay if risk is realized.
+
+Maximum Delay (Months)
+	Maximum delay if risk is realized.
+
+Likely Delay (Months)
+	Most likely delay if risk is realized.
+
+Expected Schedule Delay (Months)
+	Automatically generated value based on ``Minimum Delay``, ``Maximum Delay``, and ``Likely Delay``.
+
+.. todo::
+   Define formula for Expected Schedule Delay.
+
+Impact Time
+	Date when realized risk would impact the schedule.
+
+Impacted Event/Milestone
+	The event or milestone of a realized risk.
+
+Basis of Estimate
+	Reference to basis of estimate capturing impact of realized risk.
+
+Minimum Cost (US Dollars)
+	Minimum cost of realized risk.
+
+Maximum Cost (US Dollars)
+	Maximum cost of realized risk.
+
+Likely Cost (US Dollars)
+	Likely cost of realized risk.
+
+Costs should be estimated as they would occur, i.e. summed over the likely time period of impact (and in approximate then-year dollars). Cost estimates need only be precise to the nearest $1M (although higher precision is appreciated). This is because your cost estimates will be multiplied by your likelihood estimates, and these are likely to be uncertain by at least a factor of two.
```

Comment:
```suggestion
Costs should be estimated as they would occur, i.e. on an approximate, time-averaged, annual basis over the likely time period of impact (and in approximate then-year dollars). This is because Rubin needs to know how much funding to hold in reserve _each year_ in order to address risks as they are realized.

Cost estimates need only be precise to the nearest $1M (although higher precision is appreciated). This is because your cost estimates will be multiplied by your likelihood estimates, and these are likely to be uncertain by at least a factor of two.
```

Reply:
Added to next commit.